src/mbed_tools/cli/main.py

```diff
@@ -15,7 +15,7 @@
 
 from mbed_tools.cli.configure import configure
 from mbed_tools.cli.list_connected_devices import list_connected_devices
-from mbed_tools.cli.project_management import new, clone, deploy, libs
+from mbed_tools.cli.project_management import new, clone, deploy
```

Comment:
Commit "project: Remove libs command" creates a need for another PR to mbed-os-5-docs to update the documentation on "how to list dependencies": https://github.com/ARMmbed/mbed-os-5-docs/blob/development/docs/tools/mbed_cli_2/use.md#list-an-applications-library-dependencies

Reply:
Added a Jira task to update the docs.